sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/query.sql

```diff
@@ -32,18 +29,19 @@ SELECT
     AND NOT `moz-fx-data-shared-prod.udf.ga_is_mozilla_browser`(device.web_info.browser)
   ) AS non_fx_downloads,
 FROM
-  `moz-fx-data-marketing-prod.analytics_313696158.events_*` a
+  `moz-fx-data-marketing-prod.analytics_313696158.events_*` AS a
 JOIN
-  UNNEST(event_params) e
+  UNNEST(event_params) AS e
 WHERE
   _TABLE_SUFFIX = FORMAT_DATE('%Y%m%d', @submission_date)
   AND e.key = 'ga_session_id'
+  AND a.event_name = 'product_download'
```

Comment:
With this change you can also remove the `event_name = 'product_download'` conditions from the counts.

Reply:
Thanks, updated!